releases/release-1.20/meeting-updates/ci-status/rel-full.md

```diff
@@ -0,0 +1,1011 @@
+# [sig-release-master-blocking](https://testgrid.k8s.io/sig-release-master-blocking) at Nov 19, 2020 18:04 UTC
+Generated by [flake-tracker](https://github.com/RobertKielty/flake-tracker)
+Out of 19 jobs <span style="background-color:#0f0;foreground-color:#555;padding: 3px;">**PASSING 12**</span> <span style="background-color:#f00;color:#fff;padding: 3px;">**FAILING 1**</span> <span style="background-color:#609;color:#fff;padding: 3px;">**FLAKING 6**</span>
+## Passing Jobs
+<details>
+<summary>Expand 12 passing Jobs</summary>
+ [Conformance - GCE - master]()  [bazel-build-master]()  [build-master]()  [build-master-fast]()  [conformance-ga-only]()  [gce-cos-master-alpha-features]()  [gce-cos-master-scalability-100]()  [gci-gce-ingress]()  [integration-master]()  [node-kubelet-master]()  [skew-cluster-latest-kubectl-stable1-gce]()  [verify-master]() 
```

Comment:
Should the passing jobs have links as well?

Reply:
Yes it will in future reports. That's a bug I have to fix.